what-is-chalk.mdx

```diff
@@ -18,215 +18,467 @@ import { PyDiffEditor } from '@/components/Editor'
 
 <DocsHero />
 
-Chalk is a feature store that enables innovative machine learning teams
-to focus on building the unique products and models that make their
-business stand out. Behind the scenes, Chalk seamlessly handles
-data infrastructure with a best-in-class developer experience.
-Here’s how it works.
+Build, deploy, and iterate faster with Chalk — a programmable feature engine that powers low-latency inference, rapid model iteration, and observability across your model lifecycle.
+Chalk solves core pain points teams face when building enterprise AI and ML systems:
 
----
+- deploying and scaling production-grade infrastructure
+- authoring feature pipelines in pure Python — No DSLs! No rewrites!
+- building training datasets with high throughput batch offline queries and point-in-time correctness
+- evaluating LLMs at scale by integrating unstructured inputs into pipelines
+- serving fresh features on-the-fly with versioning, branching, and full observability
+    - gradually rollout, easily rollback, and feature flag models with version control
+    - collaborate across teams with branch-based QA
+    - A/B test with real historical production traffic by leveraging Chalk’s feature store
 
-## Develop
+---
 
-Chalk makes it simple to develop feature pipelines for machine
-learning. Define Python functions using the libraries and tools you're
-familiar with instead of specialized DSLs. Chalk then orchestrates
-your functions into pipelines that execute in parallel on a Rust-based
-engine and coordinates the infrastructure required to compute
-features.
+## How does Chalk make feature engineering easier?
 
-### Features
+Define features using Pythonic classes in Chalk — no DSLs required.
+Every feature is:
+- typed and validated
+- versioned and testable
+- composable and reusable in both training and online inference
 
-To get started, [define your features](/docs/features) with
-[Pydantic](https://pydantic-docs.helpmanual.io/)-inspired Python classes.
-You can define schemas, specify relationships, and add metadata
-to help your team share and re-use work.
+Describe relationships between entities (e.g., users and transactions) with simple type annotations, and Chalk takes care of joins, lineage, and query planning automatically.
 
 ```py
-from chalk import _, DataFrame
 import chalk.functions as F
-from chalk.features import features
+from chalk import features, DataFrame, Windowed, windowed, _
```

Comment:
we need to `from chalk.features import features`. as written this is importing the module and treating the module as a decorator, which is going to be a bad time